tests/Unit/TemperatureTest.php

```diff
@@ -57,7 +57,6 @@ public function testCelsiusToRomer()
     public function testRomerToRankine()
     {
         $tempUnit = new Temperature\Romer(50);
-        $this->assertEquals('637.38 °R', Temperature\Rankine::from($tempUnit)->format(2));
+        $this->assertEquals('637.38 °R', Temperature\Rankine::from($tempUnit)->format(2))
```

Comment:
There is a `;` missing here (tip: run `composer cs` and/or `composer test` to run style checker and/or tests)

Reply:
```suggestion
        $this->assertEquals('637.38 °R', Temperature\Rankine::from($tempUnit)->format(2));
```
Thanks for the hint that you wrote composer scripts for that. I saw the phpcs installation but I am more used to php-cs-fixer :see_no_evil: